src/templates/layouts/layout.hbs

```diff
@@ -10,6 +10,8 @@
 
   {{> nav}}
 
+  <div id="offline-status">No internet connection detected</div>
```

Comment:
Can this be styled so it doesn't cause the content to be shifted down. A toast or flash message type of thing.

Reply:
Happy with update.